backend/open_webui/utils/plugin.py

```
@@ -166,6 +166,62 @@ def load_function_module_by_id(function_id, content=None):
         os.unlink(temp_file.name)
 
 
+def get_function_module_from_cache(request, function_id, load_from_db=True):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_function_module_from_cache`

(ANN201)